extensions/rich_text_components/Image/webdriverio.js

```diff
@@ -0,0 +1,189 @@
+    '<g style=""><g transform="matrix(1 0 0 1 0 0)" id="group1"><circle styl' +
+    'e="stroke: rgb(0,128,0); stroke-width: 4; stroke-dasharray: none; strok' +
+    'e-linecap: butt; stroke-dashoffset: 0; stroke-linejoin: miter; stroke-m' +
+    'iterlimit: 4; fill: rgb(255,255,0); fill-rule: nonzero; opacity: 1;" cx' +
+    '="0" cy="0" r="40" id="group1"/></g></g></g></svg>')
+};
+
+var customizeComponent = async function(modal, selectedAction, args, altText) {
+  // The arguments for this function are as follows:
+  //   - modal: The selector for the customisation arguments modal.
+  //   - selectedAction: Represents the user action. Image RTE component
+  //       allows uploading an image from the system or creation of an
+  //       image from scratch using the diagram editor. This argument
+  //       takes the value 'create' or 'upload'.
+  //   - args: The input to the editor. If the selected action is 'create',
+  //       args would be a list of shapes and for 'upload' action, it
+  //       would represent a filepath.
+  //   - altText: The alt text for the image.
+  if (selectedAction === 'create') {
+    await action.click(
+      'Create button',
+      modal.$('.e2e-test-create-image'));
+    for (var i = 0; i < args.length; i++) {
+      var shapeClass = '.e2e-test-create-' + args[i];
+      var shapeTool = modal.$(shapeClass);
+      var customTools = ['bezier', 'piechart', 'svgupload'];
+      if (customTools.includes(args[i])) {
+        await action.click('Shape tool', shapeTool);
+        if (args[i] === 'svgupload') {
+          var imageUploadInput = $('.e2e-test-photo-upload-input');
+          absPath = path.resolve(__dirname, './circle.svg');
+          await waitFor.presenceOf(
+            imageUploadInput,
+            'Image upload input element took too long to load');
+          await imageUploadInput.keys(absPath);
+        }
+      }
+      await action.click('Shape tool', shapeTool);
+    }
+    var altTextInputElement = (
+      $(
+        '[placeholder = "Description of Image (Example : George Handel, ' +
+        '18th century baroque composer)"]'));
+    var saveDiagram = modal.$('.e2e-test-save-diagram');
+    await action.click('Save diagram button', saveDiagram);
+    await waitFor.visibilityOf(
+      modal.$('.e2e-test-saved-diagram-container'),
+      'Diagram container not visible');
+    await action.keys('Alt text input', altTextInputElement, altText);
+  }
+};
+
+var expectComponentDetailsToMatch = async function(
+    elem, selectedAction, args, altText) {
+  if (selectedAction === 'create') {
+    var svgName = args.join('_');
+    var svgDiagramInputElement = elem.$('.e2e-test-image');
+    await waitFor.visibilityOf(
+      svgDiagramInputElement,
+      'SVG Diagram input element takes too long to load.');
+    var src = await svgDiagramInputElement.getAttribute('src');
+    var alt = await svgDiagramInputElement.getAttribute('alt');
+    aexpect(alt).toEqual(altText);
```

Comment:
```suggestion
    expect(alt).toEqual(altText);
```
But how it even passed on CI??

Reply:
Updated. This code is not yet used thats why it passed